permissions/component-icon-set.yml

```diff
@@ -5,4 +5,3 @@ paths:
 - "org/jenkins-ci/plugins/icon-shim/icon-set"
 developers:
 - "jglick"
```

Comment:
IIRC this is dead anyway @timja? Is the repo archived? I wonder if we should delete permission files for such repos.

Reply:
oops, archived now, could do, no strong opinion

> Your repository "jenkinsci/icon-shim-plugin" was successfully archived.